ext/fg/js/api.js

```diff
@@ -53,6 +53,10 @@ function apiCommandExec(command) {
     return utilInvoke('commandExec', {command});
 }
 
-function apiAudioGetUrl(definition, source) {
```

Comment:
I guess this was accidentally removed when adding `apiScreenshotGet` and `apiForward`?

Reply:
Yup... just re-added that.

Reply:
#183